qt/scientific_interfaces/General/StepScan.cpp

```diff
@@ -548,11 +548,16 @@ bool StepScan::runStepScanAlgLive(std::string stepScanProperties) {
 
   Json::Value root;
   Json::Reader reader;
-  bool parsingSuccessful = reader.parse(stepScanProperties.c_str(), root);
+
+  bool parsingSuccessful = reader.parse(stepScanProperties, root);
   if (!parsingSuccessful) {
     throw std::runtime_error("Parsing parameters failed for StepScan.");
   }
-  std::string ssp = root.get("properties", "").toStyledString();
+  Json::Value &prop = root["properties"];
+  if (prop.type() == Json::nullValue) {
```

Comment:
This can be reduced to `prop.isNull()`. 

Looks like it can soon be simplified even further.
https://github.com/open-source-parsers/jsoncpp/pull/695/files